docs/source/driver/snowflake.rst

```diff
@@ -446,6 +446,9 @@ These options map 1:1 with the Snowflake `Config object <https://pkg.go.dev/gith
     disabled by setting this to ``true``. Value should be either ``true``
     or ``false``.
 
+``adbc.snowflake.sql.client_option.config_file``
+    Specifies the location of the client configuration json file.
```

Comment:
```suggestion
    Specifies the location of the client configuration JSON file.
```

Is there any Snowflake doc page we can link here?

Reply:
added